.clj-kondo/config.edn

```diff
@@ -379,6 +379,7 @@
     metabase.request               #{metabase.request.core}
     metabase.sample-data           #{metabase.sample-data}
     metabase.search                #{metabase.search.core
+                                     metabase.search.init
```

Comment:
There was some search-specific stuff defined in `metabase.models` which was otherwise mostly just a namespace that collected together a lot of different model namespaces... I moved this into the `search` module itself and gave it an init namespace that gets loaded on launch.